benchmarks/CRAM.md

```diff
@@ -4,255 +4,284 @@ layout: default
 title: CRAM Benchmarks
 highlighting: yes
 ---
-CRAM benchmarking
-=================
+CRAM 3.1 becomes the default CRAM output version
+================================================
 
-(Updated Sep 2019)
+Release 1.22 of HTSlib and SAMtools has switched to writing CRAM 3.1
+by default.  Read support CRAM 3.1 has been available since HTSlib
+1.12 (March 2021), and we announced our switch to making it the
+default in September 2024.
 
-<!-- Tested on my gen1c (Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz)
+Support for CRAM 3.1 exists in other tools, including the Noodles
+(Rust) and at the time of writing a GitHub pull-request exists for
+adding it to HTSJDK.
+
+If you need to continue using CRAM 3.0, please explicitly specify the
+version when using SAMtools.  For example:
+
+    samtools view -O cram,version=3.0 file.bam -o file.cram
+
+This is also the way you can control the compression levels and/or
+profiles.  E.g.
+
+    samtools view -O cram,small,level=7 file.bam -o file.cram
+
+
+Note not all machine types may benefit from CRAM 3.1, as it largely
+depends on the randomless of the data.  However some platforms may
```

Comment:
Should this be randomness?  Apparently randomless is a word but I am not sure it makes sense here.

Reply:
Fixed and force pushed over the commit.

Thanks